test/Microsoft.AspNet.Mvc.FunctionalTests/compiler/resources/MvcTagHelpersWebSite.MvcTagHelper_Customer.Index.html

```diff
@@ -7,7 +7,7 @@
         <div>
             <label class="order" for="Number">Number</label>
             <input class="form-control input-validation-error" type="number" data-val="true" data-val-range="The field Number must be between 1 and 100." data-val-range-max="100" data-val-range-min="1" id="Number" name="Number" value="" />
-            <span class="field-validation-error" data-valmsg-for="Number" data-valmsg-replace="true">A value is required.</span>
+            <span class="field-validation-error" data-valmsg-for="Number" data-valmsg-replace="true">The value &#x27;&#x27; is invalid.</span>
```

Comment:
This is an intentional behavior change here. This code was relying on 'on the wire' semantics for a value type.